app/static/ie-status.json

```diff
@@ -1885,6 +1885,24 @@
     "id": 6049512976023552
   },
   {
+    "name": "WebGL 2.0",
+    "category": "Graphics",
+    "link": "https://www.khronos.org/registry/webgl/specs/latest/2.0/",
+    "summary": "An update to WebGL, based on OpenGL ES 3.0",
+    "standardStatus": "Working draft or equivalent",
+    "ieStatus": {
+      "text": "Under Consideration ",
+      "iePrefixed": "",
+      "ieUnprefixed": "",
+      "priority": "medium"
+    },
+    "msdn": "",
+    "wpd": "",
+    "demo": "https://www.youtube.com/watch?v=2v6iLpY7j5M",
```

Comment:
I noticed there was [another link to a youtube video](https://github.com/patrickkettner/Status/blob/27496c1ff9544a42acf716d3b4784558ecaaf741/app/static/ie-status.json#L1901), for what thats worth